### ISSUE TITLE
Cannot read property 'state of undefined

### ISSUE THREAD

**Issue by Dadbo** [kpsuperplane/homebridge-wiz-lan#58]
Recently some of my wiz bulbs have stopped responding. Here is the console output:

```
This plugin threw an error from the characteristic 'Color Temperature': Unhandled error thrown inside write handler for characteristic: Cannot read property 'state' of undefined. See https://git.io/JtMGR for more info.
[5/10/2021, 09:14:32] [homebridge-wiz-lan] TypeError: Cannot read property 'state' of undefined
    at Object.setPilot (/usr/local/lib/node_modules/homebridge-wiz-lan/src/accessories/WizBulb/pilot.ts:140:21)
    at Characteristic.<anonymous> (/usr/local/lib/node_modules/homebridge-wiz-lan/src/accessories/WizBulb/characteristics/temperature.ts:39:9)
    at Characteristic.emit (events.js:327:22)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1632:16
    at new Promise (<anonymous>)
    at Characteristic.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1630:14)
    at step (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:143:27)
    at Object.next (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:124:57)
    at /usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:117:75
    at new Promise (<anonymous>)
```
No changes on my end. 2 of my 3 wiz bulbs have stopped responding

**Comment by Dadbo** [comment]
Thanks for the suggestions. I'll give that a try.

**Comment by vipul261** [comment]
@Dadbo Were you able to fix the issue? I tried @MoTechnicalities's suggestions and added my device reserved ip directly in the config and made Wiz a child hub but I'm still getting the same error

**Comment by MoTechnicalities** [comment]
Just chiming in to let you know my setup is still working great, using HomeBridge. But also, be sure to check that everything is up to date regarding your Node.js and Npm versions.

**Comment by vipul261** [comment]
Working fine for me also now, @Dadbo  I followed the following steps in case you are still facing issues:

1. Ensure everything is updated including wiz plugin and homebridge.
2. Added bulbs manually with their reserved IPs
2. Ran Wiz as a child Hub
3. Removed Wiz Platform and re-added the complete config in homebridge.
4. Added wiz child hub again in homekit.

Everything is working perfectly fine after this. Thanks @MoTechnicalities :)
